governance.md

```diff
@@ -15,6 +15,14 @@ https://github.com/pangeo-data/governance
 
 #### Conflict of Interest
 
+It is expected that Council Members will be employed at a wide range of companies, universities and non-profit organizations. Because of this, it is possible that Members will have conflict of interests. Such conflict of interests include, but are not limited to:
+
+* Financial interests, such as investments, employment, or contracting work, outside of The Project that may influence their work on The Project.
+* Access to proprietary information of their employer that could potentially leak into their work with the Project.
+* An issue where the person privately gains an advantage from The Project resources, but The Project has no gain or suffers a disadvantage.
+
```

Comment:
is `The Project` defined elsewhere as a legaleze proper noun?

Reply:
Yes. This is defined in the Project section (see #6)